src/main/scala/io/iohk/ethereum/network/p2p/MessageSerializableImplicit.scala

```diff
@@ -0,0 +1,15 @@
+package io.iohk.ethereum.network.p2p
+
+/**
+  * Helper class
+  */
+abstract class MessageSerializableImplicit[T <: Message](val msg: T) extends MessageSerializable {
```

Comment:
oh I see, and since we no longer use type-class like mechanism, we need those, right?